packages/@aws-cdk/aws-cloudtrail/lib/index.ts

```diff
@@ -150,7 +150,7 @@ export class CloudTrail extends cdk.Construct {
 
       logsRole = new iam.Role(this, 'LogsRole', { assumedBy: new iam.ServicePrincipal(cloudTrailPrincipal) });
 
-      const streamArn = `${logsRole.roleArn}:log-stream:*`;
+      const streamArn = `${logGroup.logGroupArn}:*`;
```

Comment:
Why did you drop `log-stream`?

Reply:
Can you confirm that the `:*` suffix is actually necessary? I believe the `logGroupArn` will already end in `:*`.

Reply:
`log-stream` was dropped as the cloud trail fails to deploy with it. The reason been that the `logs:CreateLogStream` action should be on the log group resource and not on the (as yet to be created) log stream. I think the `logs:PutLogEvents` action could be scoped to just the log stream if we want to be stricter about the permissions.

I tried without `:*` and it works fine. I'll make the change.